Sources/Sentry/SentryClient.m

```diff
@@ -113,6 +114,10 @@ - (instancetype)initWithOptions:(SentryOptions *)options
         self.locale = locale;
         self.timezone = timezone;
         self.attachmentProcessors = [[NSMutableArray alloc] init];
+        self.logBatcher = [[SentryLogBatcher alloc]
+            initWithOptions:options
+              dispatchQueue:SentryDependencyContainer.sharedInstance.dispatchQueueWrapper];
+        self.logBatcher.delegate = self;
```

Comment:
`m`: I think we could already pass the `SentryLogBatcherDelegate` in via the init method and make the property private to the `SentryLogBatcher`.